### DISCUSSION TITLE
Enable https module does not seem to redirect to https

### DISCUSSION THREAD

**Discussion by giannik** [OrchardCMS/OrchardCore#11082] [Q&A]
I enabled the https module and set it up as shown below to redirect all requests to https .

But when going to frontend and calling home page without https it doesnt seem to redirect to https and browser says site its not secure.

Certificates are all ok , if i manually type in https it works , but the automatic redirect from http does not work.

Is there something I might be missing ?  

![image](https://user-images.githubusercontent.com/1516981/151385194-b1c45789-6d11-4171-848d-777a990aef10.png)

**Comment by Skrypt** [comment]
Moved to : https://github.com/OrchardCMS/OrchardCore/issues/11085
